.travis.yml

```diff
@@ -13,5 +13,6 @@ cache:
 install: true
 
 script:
+  - if [[ $TRAVIS_JDK_VERSION != "openjdk6" ]]; then ./mvn-build-all.sh xml:check-format formatter:validate; fi
```

Comment:
- [ ] It's not optimally.
`formatter:validate` is not work using jdk 6, but `xml:check-format` work.
See https://travis-ci.org/terasolunaorg/terasoluna-gfw/jobs/251028127#L656 .